js/ftx.js

```diff
@@ -1925,6 +1927,31 @@ module.exports = class ftx extends Exchange {
         return result;
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name ftx#cancelOrders
+         * @description cancel multiple orders
+         * @param {[string]} ids order ids
+         * @param {string|undefined} symbol not used by ftx cancelOrders ()
+         * @param {object} params extra parameters specific to the ftx api endpoint
+         * @returns {object} a list of order ids queued for cancelation
+         */
+        await this.loadMarkets ();
+        const request = {
+            'orderIds': ids,
+        };
+        const response = await this.privateDeleteBulkOrders (this.extend (request, params));
+        //
+        //     {
+        //         "success": true,
+        //         "result": "Order ids queued for cancelation"
+        //     }
+        //
+        const result = this.safeValue (response, 'result', []);
+        return result;
```

Comment:
If we're not returning a unified response, we should just return the entire raw object